package.json

```diff
@@ -22,7 +23,11 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "lint": "eslint src/**/*.{js,jsx,ts,tsx,json}",
```

Comment:
While the code base is getting bigger, eslint scripts take longer to run. To optimize this we can use `lint-staged` https://www.npmjs.com/package/lint-staged